dnf-behave-tests/dnf/remove-no-provides.feature

```diff
@@ -0,0 +1,27 @@
+@dnf5daemon
+Feature: Remove RPMs by provides
+
+
+Background: Install glibc
```

Comment:
The Backround text is wrong. It should read "Install two packages, both providing foo".

Reply:
Thank you, fixed